CMakeLists.txt

```diff
@@ -50,8 +46,13 @@ else()
     include(ecbuild_log) #Required for ecbuild/FindNetCDF.cmake
 endif()
 
+find_package(eckit QUIET)
+find_package(oops QUIET)
```

Comment:
If I understood the purpose of `jedi-base` was to remove the need for `oops`?

Reply:
When jedi-base has those, we can remove oops.

Reply:
Looks like jedi-base was not selected for release. So, use oops for now.

Reply:
Thanks for the confirmation @rhoneyager